.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,6 @@
 -   id: autopep8-wrapper
-    name: autopep8 wrapper
-    description: "Runs autopep8 over python source.  If you configure additional arguments you'll want to at least include -i."
+    name: autopep8 wrapper (DEPRECATED, use pre-commit/mirrors-autopep8)
+    description: "Runs autopep8 over python source.  If you configure additional arguments you'll want to at least include -i. (This is deprecated, use pre-commit/mirrors-autopep8)."
```

Comment:
Let's remove the old description, it's not relevant now.

Adjusting the name isn't necessary, let's leave that